### DISCUSSION TITLE
Nice stream server！

### DISCUSSION THREAD

**Discussion by x931890193** [bluenviron/mediamtx#522] [Questions]
How does it compare with [SRS](https://github.com/ossrs/srs)

**Comment by aler9** [comment]
<i>rtsp-simple-server</i>:

* is both a server and a proxy, and it can ingest existing streams from other servers and redistribute them, while SRS is only a server
* is built around the RTSP protocol, which is faster than RTMP, but also supports RTMP
* is distributed in binary form for all major OSs and architectures, while SRS is not

SRS on the other hand:

* supports the SRT protocol and has an integrated WebRTC transcoder
* has a load balancing mechanism that allows to distribute the load

In my opinion they cover different areas of application, you have to choose by evaluating what you need.

[a comment by an automated account is not shown]
